styleguide/source/assets/scss/01-atoms/_button.scss

```diff
@@ -0,0 +1,15 @@
+%ama__button,
+.ama__button,
+button,
+.ama__button--solid {
```

Comment:
If the default button is solid I would leave this as just `.ama__button` and only modify in the variant for outline (`.ama__button--outline`). If everything has a modifier than nothing in the block.